packages/web-ui/src/sections/Settings/WorkspaceName/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import { Workspace } from '@latitude-data/core'
+import { Input, Text } from '$ui/ds/atoms'
+import { SessionWorkspace } from '$ui/providers'
+import { useDebouncedCallback } from 'use-debounce'
+
+export default function WorkspaceName({
+  workspace,
+  update,
+}: {
+  workspace: SessionWorkspace
+  update: (payload: { name: string }) => Promise<Workspace>
+}) {
+  const onChange = async (ev: React.ChangeEvent<HTMLInputElement>) => {
+    const name = ev.target.value
+    if (!name) return
+
+    return update({ name: ev.target.value })
+  }
+  const debouncedChange = useDebouncedCallback(onChange, 500, {
+    trailing: true,
+  })
+
+  // TODO: i18n
```

Comment:
Really?

Reply:
it adds little friction and adding i18n later is a massive pita

Reply:
But this is an app for people who are prompting in English. Why would it need to be translated into other languages? 

Adding i18n is always a pita. Less now than later but still maintaining translations is a burden that we might avoid in this app. My 2 cents